Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "acl-2.2.53.tar.gz" "acl" "host-acl" "wrap-xattr-calls" "\
        libacl/acl_extended_file_nofollow.c \
        libacl/acl_extended_file.c \
        libacl/acl_extended_fd.c \
        libacl/acl_set_file.c \
        libacl/acl_set_fd.c \
        libacl/acl_get_file.c \
        libacl/acl_get_fd.c \
        libacl/acl_delete_def_file.c \
    ";


===============================================================================
--- a/libacl/acl_extended_file_nofollow.c	2016-02-15 18:42:44.000000000 +0100
+++ b/libacl/acl_extended_file_nofollow.c	2019-01-02 02:18:54.000000000 +0100
@@ -30,5 +30,5 @@
 int
 acl_extended_file_nofollow(const char *path_p)
 {
-	return __acl_extended_file(path_p, lgetxattr);
+	return __acl_extended_file(path_p, _lgetxattr);
 }
===============================================================================
--- a/libacl/acl_extended_file.c	2016-02-15 18:42:44.000000000 +0100
+++ b/libacl/acl_extended_file.c	2019-01-02 02:18:33.000000000 +0100
@@ -30,6 +30,6 @@
 int
 acl_extended_file(const char *path_p)
 {
-	return __acl_extended_file(path_p, getxattr);
+	return __acl_extended_file(path_p, _getxattr);
 }
 
===============================================================================
--- a/libacl/acl_extended_fd.c	2016-02-15 18:42:44.000000000 +0100
+++ b/libacl/acl_extended_fd.c	2019-01-02 02:18:00.000000000 +0100
@@ -33,12 +33,12 @@
 	int base_size = sizeof(acl_ea_header) + 3 * sizeof(acl_ea_entry);
 	int retval;
 
-	retval = fgetxattr(fd, ACL_EA_ACCESS, NULL, 0);
+	retval = _fgetxattr(fd, ACL_EA_ACCESS, NULL, 0);
 	if (retval < 0 && errno != ENOATTR && errno != ENODATA)
 		return -1;
 	if (retval > base_size)
 		return 1;
-	retval = fgetxattr(fd, ACL_EA_DEFAULT, NULL, 0);
+	retval = _fgetxattr(fd, ACL_EA_DEFAULT, NULL, 0);
 	if (retval < 0 && errno != ENOATTR && errno != ENODATA)
 		return -1;
 	if (retval >= base_size)
===============================================================================
--- a/libacl/acl_set_file.c	2016-02-15 18:42:44.000000000 +0100
+++ b/libacl/acl_set_file.c	2019-01-02 02:17:26.000000000 +0100
@@ -58,7 +58,7 @@
 	ext_acl_p = __acl_to_xattr(acl_obj_p, &size);
 	if (!ext_acl_p)
 		return -1;
-	error = setxattr(path_p, name, (char *)ext_acl_p, size, 0);
+	error = _setxattr(path_p, name, (char *)ext_acl_p, size, 0);
 	free(ext_acl_p);
 	return error;
 }
===============================================================================
--- a/libacl/acl_set_fd.c	2016-02-15 18:42:44.000000000 +0100
+++ b/libacl/acl_set_fd.c	2019-01-02 02:17:00.000000000 +0100
@@ -44,7 +44,7 @@
 	ext_acl_p = __acl_to_xattr(acl_obj_p, &size);
 	if (!ext_acl_p)
 		return -1;
-	error = fsetxattr(fd, name, (char *)ext_acl_p, size, 0);
+	error = _fsetxattr(fd, name, (char *)ext_acl_p, size, 0);
 	free(ext_acl_p);
 	return error;
 }
===============================================================================
--- a/libacl/acl_get_file.c	2016-02-15 20:13:51.000000000 +0100
+++ b/libacl/acl_get_file.c	2019-01-02 02:15:34.000000000 +0100
@@ -54,14 +54,14 @@
 
 	if (!ext_acl_p)
 		return NULL;
-	retval = getxattr(path_p, name, ext_acl_p, size_guess);
+	retval = _getxattr(path_p, name, ext_acl_p, size_guess);
 	if (retval == -1 && errno == ERANGE) {
-		retval = getxattr(path_p, name, NULL, 0);
+		retval = _getxattr(path_p, name, NULL, 0);
 		if (retval > 0) {
 			ext_acl_p = alloca(retval);
 			if (!ext_acl_p)
 				return NULL;
-			retval = getxattr(path_p, name, ext_acl_p, retval);
+			retval = _getxattr(path_p, name, ext_acl_p, retval);
 		}
 	}
 	if (retval > 0) {
===============================================================================
--- a/libacl/acl_get_fd.c	2016-02-15 20:13:51.000000000 +0100
+++ b/libacl/acl_get_fd.c	2019-01-02 02:14:46.000000000 +0100
@@ -41,14 +41,14 @@
 
 	if (!ext_acl_p)
 		return NULL;
-	retval = fgetxattr(fd, ACL_EA_ACCESS, ext_acl_p, size_guess);
+	retval = _fgetxattr(fd, ACL_EA_ACCESS, ext_acl_p, size_guess);
 	if (retval == -1 && errno == ERANGE) {
-		retval = fgetxattr(fd, ACL_EA_ACCESS, NULL, 0);
+		retval = _fgetxattr(fd, ACL_EA_ACCESS, NULL, 0);
 		if (retval > 0) {
 			ext_acl_p = alloca(retval);
 			if (!ext_acl_p)
 				return NULL;
-			retval = fgetxattr(fd, ACL_EA_ACCESS, ext_acl_p,retval);
+			retval = _fgetxattr(fd, ACL_EA_ACCESS, ext_acl_p,retval);
 		}
 	}
 	if (retval > 0) {
===============================================================================
--- a/libacl/acl_delete_def_file.c	2016-02-15 18:42:44.000000000 +0100
+++ b/libacl/acl_delete_def_file.c	2019-01-02 02:13:06.000000000 +0100
@@ -32,7 +32,7 @@
 {
 	int error;
 	
-	error = removexattr(path_p, ACL_EA_DEFAULT);
+	error = _removexattr(path_p, ACL_EA_DEFAULT);
 	if (error < 0 && errno != ENOATTR && errno != ENODATA)
 		return -1;
 	return 0;
